models/sipnet/R/write.configs.SIPNET.R

```diff
@@ -45,6 +45,7 @@ write.config.SIPNET <- function(defaults, trait.values, settings, run.id, inputs
   } else {
     jobsh <- readLines(con = system.file("template.job", package = "PEcAn.SIPNET"), n = -1)
   }
+  jobsh[1] <- paste(jobsh[1],"-l")
```

Comment:
Am I right that this makes the first line of the generated job.sh be `#!/bin/bash -l` (where `-l` indicates "login mode" and causes bash to read user profile files on startup) ? 

If so, I'm weakly inclined against this: The standard advice for shell scripting is to keep the script self-contained and to avoid chances for the user's personal configuration to affect script behavior. If you need to provide specific settings to be able to use sipnet on your machine, I think it's probably better to pass them in specifically -- perhaps in settings$model$prerun?

This is a tentative opinion, though -- I'm open to counterarguments.

Reply:
yes! should I just grab info from the setting object and create a if statement like that:
if(is geo){
jobsh[1] <- paste(jobsh[1],"-l")
}
such that, it won't break anything if it's not on geo.

Reply:
I haven't figured out how to pass this info to the pre-model run section.

Reply:
What specifically needs to be passed in?

Reply:
@DongchenZ what specifically needs to be set from your profile for the script to run? Is there a reason that can't be passed in using the `@HOST_SETUP@` tag in the template job https://github.com/PecanProject/pecan/blob/develop/models/sipnet/inst/template.job, which maps onto `settings$model$prerun`